extensions/arm/cl_arm_scheduling_controls.html

```diff
@@ -937,7 +970,14 @@ <h2 id="_modifications_to_the_opencl_api_specification"><a class="anchor" href="
 - <code>CL_DEVICE_SCHEDULING_DEFERRED_FLUSH_ARM</code> is set when the device supports
   <code>CL_QUEUE_DEFERRED_FLUSH_ARM</code>.</p>
 <p class="tableblock">- <code>CL_DEVICE_SCHEDULING_REGISTER_ALLOCATION_ARM</code> is set when the device compiler
-  supports the <code>-fregister-allocation</code> option.</p></td>
+  supports the <code>-fregister-allocation</code> option.</p>
+<p class="tableblock">- <code>CL_DEVICE_<wbr>SCHEDULING_<wbr>WARP_<wbr>THROTTLING_<wbr>ARM</code> is set when the device supports
+  <code>CL_KERNEL_<wbr>EXEC_<wbr>INFO_<wbr>WARP_<wbr>COUNT_<wbr>LIMIT_<wbr>ARM</code>, <code>CL_KERNEL_<wbr>MAX_<wbr>WARP_<wbr>COUNT_<wbr>ARM</code>
+  and <code>CL_DEVICE_<wbr>MAX_<wbr>WARP_<wbr>COUNT_<wbr>ARM</code>.<br>
+  Missing before version 0.4.</p>
+<p class="tableblock">- {CL_DEVICE_SCHEDULING_COMPUTE_UNIT_BATCH_QUEUE_SIZE_ARM} is set when the
```

Comment:
Looks like this enum wasn't found in the generated asciidoc dictionaries.  Are you OK posting as-is or do you want to track down what happened and rebuild the HTML spec?

Reply:
Good catch! I hadn't updated the HTML following your comment on https://github.com/KhronosGroup/OpenCL-Docs/pull/823. Now fixed. We really ought to think about automating the publication process.